page/index.html

```diff
@@ -19,6 +19,7 @@
   </head>
   <body>
     <div id="root"></div>
+    <script type="module" src="/src/main.jsx"></script>
```

Comment:
jsx extension is more for components than main file

Reply:
I can't seem to export Vite when I change the `main.jsx` file or this script tag to `.js`.
Or am I missing something?

Reply:
yes it seems not working too on my computer ... and I didn't know why